docs/pages/cheatsheet.adoc

```diff
@@ -563,12 +634,6 @@ Removed from service object.
 |resource       | access name           | The name of an access where the returned transfer object is available.
 |====================
 
-[options="header", cols="1,2"]
-|====================
-|Annotation | Meaning
-|@Delete    | Generates the default delete behavior.
-|====================
-
 === View submit
 
 *Syntax:*
```

Comment:
Can we please explain the differences on the View between `submit` and `action`? We talked about it internally, but it should be documented here as well.

Reply:
submit commented out